bin/check_tron_jobs.py

```diff
@@ -181,23 +186,15 @@ def is_action_failed_or_unknown(job_run):
 
 
 def is_job_scheduled(job_runs):
-    for run in job_runs['runs']:
-        if run.get('state', 'unknown') in ["scheduled", "queued"]:
-            return run
+    for job_run in job_runs:
+        if job_run.get('state', 'unknown') in ["scheduled", "queued"]:
+            return job_run
     return None
 
 
-def is_job_stuck(job_runs):
+def is_job_stuck(job_runs, job_expected_runtime, actions_expected_runtime):
     next_run_time = None
-
-    job_expected_runtime = job_runs.get('expected_runtime', None)
-    actions_expected_runtime = job_runs.get('actions_expected_runtime', {})
-
-    for job_run in sorted(
```

Comment:
It is interesting to me that we were already doing this. 

The only issue that I can think of is that this function now "assumes" that the job_runs object is coming in pre-sorted by time.

I think that is "OK", but could be a source of bugs in the future.

Reply:
Hmm...maybe. But sorted job_runs is most close to our assumption that is monitoring the most recent one.